cmd/inspector/main.go

```diff
@@ -110,10 +110,11 @@ var (
 
 // Inspector gives access to kademlia and overlay cache
 type Inspector struct {
-	identity      *identity.FullIdentity
-	kadclient     pb.KadInspectorClient
-	overlayclient pb.OverlayInspectorClient
-	statdbclient  pb.StatDBInspectorClient
+	transportClient transport.Client
+	identity        *identity.FullIdentity
+	kadclient       pb.KadInspectorClient
```

Comment:
extremely nitty nit but `transportClient` is camelCased but `kadclient` and the other clients aren't

Reply:
I didn't originally do the inspector struct so I'm reluctant to change an existing field name in this PR but I agree we should be consistent and be using camelCase.